docs/api/ref/schemas/image_selected.yaml

```diff
@@ -0,0 +1,78 @@
+components:
+  schemas:
+    ImageSelected:
+      type: object
+      title: Selected image
+      description: |
+        Images selected (and possibly cropped or rotated) to show an area of interest
+        (front of product, ingredients, nutrition facts, packaging info) in a specific language.
+      properties:
```

Comment:
Instead of having url directly here, we have to seek for selected_images field ?
That would be useful to explain this in the description.